test/GithubApi.Put.test.js

```diff
@@ -0,0 +1,80 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+require('dotenv').config();
+
+describe('Should interact with the user', () => {
+  const urlBase = 'https://api.github.com/user/following';
+  const user = '/aperdomob';
+
+  describe('Should follow the user', () => {
+    let followUser;
+    before(async () => {
+      followUser = await axios.put(
+        `${urlBase}${user}`,
+        {},
+        {
+          headers: {
+            Authorization: `token ${process.env.ACCESS_TOKEN}`
+          }
+        }
+      );
+    });
```

Comment:
if you want, you can create a global instance of axios with the header included, so you don't need to pass the header in each test.
```js
const instance = axios.create({
  headers: { Authorization: `token ${process.env.ACCESS_TOKEN}` }
});
```